src/robot_manager.cpp

```diff
@@ -557,6 +574,61 @@ bool RobotManager::setTargetCb(temoto_robot_manager::RobotSetTarget::Request& re
   return true;
 }
 
+
+
+bool RobotManager::getTargetCb(temoto_robot_manager::RobotGetTarget::Request& req,
+                               temoto_robot_manager::RobotGetTarget::Response& res)
+{
+  if (active_robot_->isLocal())
+  {
+    TEMOTO_INFO("Request: '%s'", req.ref_joint.c_str());
+    TEMOTO_INFO("Request: '%s'", req.respect_to.c_str());
+    res.pose = active_robot_->getTarget();
+  }
+  else
+  {
+    TEMOTO_INFO("robot is not local");
+  }
+
+  return true;
+}
+
+
+bool RobotManager::goalCb(temoto_robot_manager::RobotGoal::Request& req, temoto_robot_manager::RobotGoal::Response& res)
```

Comment:
Goal of what?

Reply:
changed to goalNavigationCb.
Is this one better?

Reply:
yes